src/Components/BlueButton/BlueButton.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import styles from "./BlueButton.module.scss";
+
+type BlueButtonProps = {
+  text: string;
+  onClick(args: React.MouseEvent<HTMLElement>): void;
```

Comment:
버튼에 굳이 MouseEvent 쓸 일이 있을까 싶습니다.
사소한 문제지만 `args` 네이밍 수정해주시면 좋을 것 같습니다.

Reply:
args 네이밍은 수정했습니다. 그런데 버튼은 클릭으로 이루어지는 것이기 때문에, MouseEvent가 맞을 것 같다고 생각했습니다. 타입스크립트라 argument의 타입도 정해줘야하는데, 구글링해봤을 때는 mouseEvent 였는데, 혹시 다른 방법이 있나요?

Reply:
굳이 이벤트를 넣는다면 MouseEvent가 맞겠지만, 클릭 이벤트의 경우는 event 객체에서 정보를 뽑아낼 일이 별로 없으니까 아예 인자를 없애도 될 것 같다는 얘기였습니다. typescript 상으로 오류도 없습니다.
```suggestion
  onClick(): void;
```